src/index.js

```diff
@@ -0,0 +1,28 @@
+import './style.css';
+import Tasks from './task.js';
+import showAllTodo from './function.js';
+// import slist from './drag.js';
+
+const todos = new Tasks();
+
+const form = document.querySelector('.add-list');
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const items = {
+    description: form.elements.todo.value,
+  };
+  form.reset();
+  todos.addTodo(items);
+  showAllTodo(todos);
+  // slist(todos);
+});
+
+showAllTodo(todos);
+
+const clear = document.querySelector('.remove');
+
```

Comment:
some recommend that all initialization must go at the start of the file. what you can do is this:

``` js
const clear;
// . . .
clear = clear = document.querySelector('.remove');
```